internal/envoy/bootstrap.go

```diff
@@ -102,6 +105,20 @@ func (c *BootstrapConfig) GetAdminAccessLogPath() string {
 func (c *BootstrapConfig) GetDNSLookupFamily() string {
 	return stringOrDefault(c.DNSLookupFamily, "auto")
 }
+
+// ValidAdminAddress checks if the address supplied is
+// "localhost" or an IP address. Only a Unix Socket
+// is supported for this address to mitigate security.
+func ValidAdminAddress(address string) error {
```

Comment:
this could also go in the `internal/validation` package as well

Reply:
Yup we could, not sure how useful it would be in other packages.